sonic_data_client/mixed_db_client.go

```diff
@@ -1133,14 +1133,16 @@ import json
 
 yang_parser = sonic_yang.SonicYang("/usr/local/yang-models")
 yang_parser.loadYangModel()
-text = '''%s'''
-
-try:
-    yang_parser.loadData(configdbJson=json.loads(text))
-    yang_parser.validate_data_tree()
-except sonic_yang.SonicYangException as e:
-    print("Yang validation error: {}".format(str(e)))
-    raise
+filename = "%s"
+with open(filename, 'r') as fp:
+	text = fp.read()
+
+	try:
+		yang_parser.loadData(configdbJson=json.loads(text))
+		yang_parser.validate_data_tree()
+	except sonic_yang.SonicYangException as e:
+		print("Yang validation error: {}".format(str(e)))
```

Comment:
Add filename to log message and log to syslog?

Reply:
filename is fixed, "/tmp/config_db.json.tmp", no need to add syslog.